packages/o/openblas/xmake.lua

```diff
@@ -29,8 +29,6 @@ package("openblas")
             add_versions("0.3.24", "92f8e0c73e1eec3c428b210fbd69b91e966f8cf1f998f3b60a52f024b2bf9d27")
             add_versions("0.3.26", "9c3d48c3c21cd2341d642a63ee8a655205587befdab46462df7e0104d6771f67")
         end
-
-        add_configs("shared", {description = "Build shared library.", default = true, type = "boolean", readonly = true})
```

Comment:
this config should be kept on windows to remind xmake that it contains only shared binaries